csi/controller.go

```diff
@@ -980,6 +1008,99 @@ func (s *OsdCsiServer) CreateSnapshot(
 		},
 	}, nil
 }
+func (s *OsdCsiServer) getCloudBackupClient() error {
+	if s.cloudBackupClient == nil {
+		// Get grpc connection
+		conn, err := s.getConn()
```

Comment:
We'll need to use `getRemoteConn` to spread the load of cloud backups across all nodes. As a result, it's best not to store this client locally in OsdCsiServer.

See this line for example:
https://github.com/libopenstorage/openstorage/blob/master/csi/controller.go#L548-L557